docs/classification/isoseq-collapse.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+parent: Classification
+title: Pigeon Workflow
+nav_order: 2
+---
+
+## IsoSeq collapse
+
+After reads are mapped to a reference genome, transcripts can be collapsed into unique isoforms using `isoseq3 collapse`. Mapped IsoSeq reads are collapsed to produce a set of unique isoforms for each locus of the genome in GFF format, along with a secondary files containing information about the number of processed reads supporting each unique isoform.
+
+### Collapse Examples
+
+<img src="../img/collapse.png" alt="collapse" width="1000px"/>
+
+### Execution
+
+Map reads using _pbmm2_ before collapsing
+
+```
+pbmm2 align --preset ISOSEQ --sort <input.bam> <ref.fa> <mapped.bam>
+```
+
+Collapse mapped reads into unique isoforms using _isoseq collapse_.
+
+```
+isoseq3 collapse <mapped.bam> <collapse.gff>
+```
+
+### Ouptut
+
+- `collapse.gff` contains the collapsed isoforms in gff format.
+- `*.abundance.txt` contains information about the number of FLNC reads supporting each isoform and cell barcodes if applicable.
```

Comment:
add:
"each unique isoform has the ID format PB.X.Y, while `count_fl` denotes the number of unique molecules (after UMI deduplication) supporting the isoform, and `fl_assoc` denotes the number of reads (before UMI deduplication) supporting it. `cell_barcodes` shows the list of single cell barcodes from which the reads came from, if applicable.